gammapy/makers/utils.py

```diff
@@ -98,6 +98,38 @@ def _get_fov_coords(pointing, irf, geom, use_region_center=True, obstime=None):
     return coords
 
 
+def time_limit_rotation(maximum_rotation, pointing_altaz):
+    """
+    Compute the maximum time such that the FoV associated to a fixed RaDec position rotates by less than
+    'maximum_rotation' in AltAz frame. It assumes that the rotation rate at the provided pointing is a good estimate
+    of the rotation rate over the full output duration (first order approximation).
+
+
+    Parameters
+    ----------
+    maximum_rotation : `~astropy.units.Quantity`
+        Maximum rotation angle.
+    pointing_altaz : `~astropy.coordinates.SkyCoord`
+        Pointing direction.
+
+    Returns
+    -------
+    duration : `~astropy.units.Quantity`
+        Time associated with the requested rotation.
+    """
+    earth_rotation_deg_per_second = 0.004167
+    denom = (
+        earth_rotation_deg_per_second
+        * np.cos(pointing_altaz.location.lat.rad)
+        * np.cos(pointing_altaz.az.rad)
+    )
+    if denom == 0:
+        return 3600 * u.s
+    return (
+        maximum_rotation.to_value(u.deg) * np.cos(pointing_altaz.alt.rad) / denom
```

Comment:
I will try to find a way to test it. But negative values are expected if one of the cosinus is negative. I think it can't happen for the telescope latitude (in [-90, 90] deg ), or for the pointing altitude (same range). But half of the azimuths should give negative values of `np.cos(pointing_altaz.az.rad)`.

If I am right it should not raise an error.

Reply:
I tested and indeed got negative values for, as a test, observations of the galactic center from CTA northern site.
I now use absolute values for `np.cos(pointing_altaz.az.rad)`